hnn_core/network.py

```diff
@@ -52,31 +55,31 @@ def _create_cell_coords(n_pyr_x, n_pyr_y, zdiff=1307.4):
     pos_dict = dict()
 
     # PYRAMIDAL CELLS
-    xxrange = np.arange(n_pyr_x)
-    yyrange = np.arange(n_pyr_y)
+    xxrange = np.arange(n_pyr_x) * inplane_distance
+    yyrange = np.arange(n_pyr_y) * inplane_distance
 
     pos_dict['L5_pyramidal'] = [
         pos for pos in it.product(xxrange, yyrange, [0])]
     pos_dict['L2_pyramidal'] = [
         pos for pos in it.product(xxrange, yyrange, [zdiff])]
 
     # BASKET CELLS
-    xzero = np.arange(0, n_pyr_x, 3)
-    xone = np.arange(1, n_pyr_x, 3)
+    xzero = np.arange(0, n_pyr_x, 3) * inplane_distance
+    xone = np.arange(1, n_pyr_x, 3) * inplane_distance
     # split even and odd y vals
-    yeven = np.arange(0, n_pyr_y, 2)
-    yodd = np.arange(1, n_pyr_y, 2)
+    yeven = np.arange(0, n_pyr_y, 2) * inplane_distance
+    yodd = np.arange(1, n_pyr_y, 2) * inplane_distance
     # create general list of x,y coords and sort it
     coords = [pos for pos in it.product(
         xzero, yeven)] + [pos for pos in it.product(xone, yodd)]
     coords_sorted = sorted(coords, key=lambda pos: pos[1])
     # append the z value for position for L2 and L5
     # print(len(coords_sorted))
 
-    pos_dict['L5_basket'] = [pos_xy + (0,) for
+    pos_dict['L5_basket'] = [(pos_xy[0], pos_xy[1], zdiff / 2) for
+                             pos_xy in coords_sorted]
+    pos_dict['L2_basket'] = [(pos_xy[0], pos_xy[1], 1.3 * zdiff) for
```

Comment:
Note that the 1.3 here is completely _ad hoc_, as is placing the L5 baskets halfway between the pyramidal layers. I'd like to hear @stephanie-r-jones take on where we should "place" the basket cells? Or more to the point: what sort of documentation is appropriate to describe the biological function modelled by them? Note that the position in Z (depth) has no impact on synaptic transmission, as 'lamtha' is purely in-plane.